src/main/cpp/src/query_operations/broad_combined_gvcf.cc

```diff
@@ -663,7 +663,16 @@ void BroadCombinedGVCFOperator::handle_FORMAT_fields(const Variant& variant) {
   //Pointer to extended vector inside field handler object
   const void* ptr = 0;
   uint64_t num_elements = 0ull;
-  int* int_vec = 0;     //will point to same address as ptr, but as int*
+  int* GT_vec = 0; // will point to same address as ptr, but as int*
+  int* DP_vec = 0; // ditto
+  //Hold values to figure out if any samples have GQ==0 and PL[0]==0 after the loop
+  //see https://github.com/broadinstitute/gatk/pull/8715
+  std::vector<bool> GQ_val(variant.get_num_calls(), false);
+  std::vector<bool> PL_val(variant.get_num_calls(), false);
+  bool some_GQ_vals_zero = false;
+  bool some_PL_first_vals_zero = false;
```

Comment:
nit: we don't seem to use these `some_*` booleans anywhere

Reply:
I am using sets now (see lines 674/675 below)and I thought I committed those changes. Will remove them, thanks!